nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -1928,7 +1928,9 @@ The following actions are not considered local changes:
 * modifying sensitive property values
 * modifying remote process group URLs
 * updating a processor that was referencing a non-existent controller service to reference an externally available controller service
-* modifying variables
+* creating or modifying variables
```

Comment:
A bit nit-picky, but how about deleting variables?

Reply:
That's a good point.  Will add that scenario.

Reply:
Added "deleting variables" scenario.  PR read for review/merge if no other issues raised. Thanks @ijokarumawak !